src/sentry/static/sentry/less/sentry-hovercard.less

```diff
@@ -30,25 +30,43 @@
   // Default align hovercard top-middle
   bottom: 28px;
   left: 50%;
-  margin-left: -@hovercard-width / 2;
+  transform: translateY(0) translateX(-50%);
   width: @hovercard-width;
   visibility: hidden;
 
   @keyframes hovercardSlideUp {
     from {
-      transform: translateY(12px);
+      transform: translateY(12px) translateX(-50%);
     }
     to {
-      transform: translateY(0);
+      transform: translateY(0) translateX(-50%);
+    }
+  }
+
+  @keyframes hovercardSlideUpNoCenter {
+    from {
+      transform: translateY(12px) translateX(0);
```

Comment:
this is kind of annoying, but currently there isn't a good way to do independent transformations other than adding a second container, which seemed objectively worse.